README.md

```diff
@@ -8,6 +8,124 @@ The website was created in [Jekyll](https://jekyllrb.com/), a static website gen
 
 The design is based on the [Libris theme](https://github.com/stackbit-themes/libris-jekyll) by [Stackbit](https://www.stackbit.com/). The theme's SASS files are stored in the `_sass/imports` folder.
 
+## Getting started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+  Install or update ruby
```

Comment:
Maybe mention that Jekyll requires Ruby version 2.5.0 or higher (easily checked using `ruby -v`).

Reply:
Yes, that makes sense. If the user has Ruby 3 'bundle exec jekyll serve' will pull in the dependency on webrick.